app/Http/Controllers/SearchController.php

```diff
@@ -15,13 +15,23 @@ public function search()
     {
         $searchInput = Input::get('searchinput');
 
+        /**
+         * During testing, the db coneection is set to sqlite
+         */
```

Comment:
**Code Quality - Nitpick** 

It should be `connection` not `coneection`.

You don't need to check if it is `pgsql`. What if we migrate to `mysql` or `oracle`? . 

A better way of doing it is this:

You can have a small method in the controller like so:

```php
private function checkSqliteConnection() {
    if(env('DB_CONNECTION') === 'sqlite') {
        return true;
    }

    return false;
}
```

Then you can call that method in this method like so:

```php
if($this->checkSqliteConnection()) {
  $like = 'LIKE';
}
```